extension/components/PageInfo.tsx

```
@@ -0,0 +1,40 @@
+import React from "react";
+import { Globe, Clock } from "lucide-react";
+import { truncateText, formatTime } from "../lib/utils";
+import type { PageInfo } from "../lib/types";
+
+interface PageInfoProps {
+  pageInfo: PageInfo | null;
+}
+
+export function PageInfo({ pageInfo }: PageInfoProps) {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'PageInfo'. Consider to delete it or rename it.

'PageInfo' is defined here:


(lint/suspicious/noRedeclare)